.github/workflows/lint-vocab.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint VC Vocabulary
+on:
+  pull_request:
+    paths:
+    - 'vocab/security/vocabulary.yml'
+    - 'vocab/security/template.html'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup environment
+        uses: actions/checkout/@v3
+      - name: Setup Node 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
```suggestion
      - name: Setup Node 20
        uses: actions/setup-node@v3
        with:
          node-version: 20
```

Reply:
@dlongley is this what you mean?

Note that this should be updated in other repositories, too!

Reply:
Yeah, that's what I meant.